src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionAddActionTest.java

```diff
@@ -697,6 +706,78 @@ public void testExecuteAndPostProcessRubricQuestion() {
                                     + "Please choose the most appropriate choices for the sub-questions below."
                                     + "|||/page/instructorFeedbackQuestionAdd";
         AssertHelper.assertLogMessageEquals(expectedLogMessage, action.getLogMessage());
+        // Clear requestedParams list
+        requestedParams.clear();
```

Comment:
This line is redundant? There are multiple places of this redundancy in your PR.